<!doctype html>
<html lang="en">

<head>
  <meta charset="utf-8">

  <title>Reveal JS presentation</title>

  <meta name="description" content="A framework for easily creating beautiful presentations using HTML">
  <meta name="author" content="Hakim El Hattab">

  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">

  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <link rel="stylesheet" href="libs/reveal.js/4.3.1/reset.css">
  <link rel="stylesheet" href="libs/reveal.js/4.3.1/reveal.css">

  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">

  <!-- highlight Theme -->
  
  <link rel="stylesheet" href="libs/highlight.js/11.3.1/styles/monokai.min.css">
  
	
		
	<link rel="stylesheet" href="libs/reveal.js/4.3.1/plugin/chalkboard/style.css">
	
	
	
	



  <!-- Revealjs Theme -->
  
  <link rel="stylesheet" href="libs/reveal.js/4.3.1/theme/black.css" id="theme">
  
  

  <link rel="stylesheet" href="libs/styles/tasklist.css">
	<link rel="stylesheet" href="libs/styles/iota.css">
	<link rel="stylesheet" href="libs/styles/layout.css">


  <!-- Revealjs Theme -->
  
  <link rel="stylesheet" href="rasi-white.css">
  

   <!-- css list -->
	

   

</head>

<body>

   

  <div class="reveal">

    <!-- Any section element inside of this container is displayed as a slide -->
    <div class="slides">

      


    
        <section >
            
            <h2>RNA-linked CRISPR Screens in Human Cells</h2>
<h3><a href="http://rasilab.org/members/arvind-rasi-subramaniam">Arvind Rasi Subramaniam</a></h3>
<p>Basic Sciences Division &amp; Herbold Computational Biology Program</p>
<div style="margin-top: 50px;font-size: 80%">
<p>22 Sep 2023</p>
</div>
<img src="./svg/patrick.png" style="width:200px"/>
<p>Patrick Nugent</p>
<aside class="notes" data-markdown="">
- Good morning everyone, and welcome to the BSD retreat.
- Today, I want to talk about a new RNA-linked CRISPR screening method that a talented graduate student, Patrick Nugent, has developed in the lab.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/mrna_lifecycle.svg" alt=""></p>
<aside class="notes" data-markdown="">
- RNAs have a complex lifecycle in cells.
- After they are transcribed, RNA are spliced and polyadenylated in the nucleus, exported into the cytosol, where they are translated and ultimately degraded.
- Hundreds of RNA-binding proteins regulate each of these steps, and these RNA-binding proteins themselves are regulated by other cellular factors and environmental cues.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/mrna_lifecycle_2.svg" alt=""></p>
<aside class="notes" data-markdown="">
- A key challenge in the field of RNA biology is to systematically define which cellular factors regulate a specific metabolic step of any given RNA.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/facs_screen.svg" alt=""></p>
<aside class="notes" data-markdown="">
- CRISPR screens are a popular way to identify genes that regulate RNA metabolism.
- Typically, an RNA metabolic step such as splicing is coupled to a fluorescent reporter,
- Cells expressing the reporter are perturbed by a genome-wide CRISPR library, and are sorted and sequenced based on the fluorescence of the reporter.
- The problem with this approach is that we are looking at protein expression, which might not necessarily change when an RNA metabolic step is perturbed, or might change for reasons that have nothing to do with RNA metabolism. So, basically you tend to have high false positives and negatives.
</aside>
            </section>
    



    
        <section >
            
            <!-- ![](svg/perturb-seq.svg)

<aside class="notes" data-markdown="">
- Alternative is to do Perturb-seq, where you do single cell RNA sequencing after CRISPR perturbations.
- But single cell screens are limited to highly expressed endogenous mRNAs, they typically sequence only 3' ends of RNAs and are not flexible, and are also really expensive to use for more than a handful of genetic perturbations.
</aside>

--- -->
<p><img src="svg/ciberseq_schematic.svg" alt=""></p>
<aside class="notes" data-markdown="">
- These limitations can be overcome by using barcoded CRISPR screens.
- Here you express a guide RNA and one or more reporters in cis. 
- A random barcode is inserted into the reporter which is linked to the guide RNA by sequencing the plasmid.
- To do the screen, you integrate the DNA into the genome, extract the RNA, and deep sequence the barcodes.
- Most guides will not have a specific effect on the reporter RNA and the counts for the corresponding two barcodes will be correlated.
- CRISPR guides that specifically increase the reporter mRNA level will have higher barcode 2 counts relative to barcode 1 counts, and vice versa.
- This approach has been done in yeast a few years back under the name CIBER-seq or reporter seq, but has not been done in human cells.
</aside>
<!-- ---

![](svg/lentivirus_template_switch.svg)

<aside class="notes" data-markdown="">
- The reason it has been challenging is that CRISPR screens in human cells are carried out using lentiviral vectors.
- These vectors integrate randomly in the genome, which causes variable expression and so cannot be used for comparing RNA levels between cells.
- Lentiviruses also undergo extensive recombination during reverse transcription, which would scramble the linkage between the reporter barcodes and the guides.
</aside>
 -->
            </section>
    



    
        <section >
            
            <p><img src="svg/bxb1_method.svg" alt=""></p>
<aside class="notes" data-markdown="">
- To implement this in human cells, Patrick used a high efficiency site-specific recombinase called Bxb1 that allows integration of the guide-reporter library into a specific genomic locus where we have already inserted an attP landing pad for the recombinase.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="./svg/relic_workflow.svg" alt=""></p>
<aside class="notes" data-markdown="">
- Using this approach, Patrick can now do RNA-linked CRISPR screens in human cells, which we call ReLiC.
- In our current iteration, we have CRISPR library targeting the 2000 RNA-binding proteins in the human genome, with 4 guide pairs per gene, and 3 random barcodes per guide pairs.
- We integrate this library into HEK293T cells, induce Cas9, and sequence the barcodes, and then look for guides that increase or decrease the reporter RNA levels.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="./svg/nmd_reporter_validation.svg" alt=""></p>
<aside class="notes" data-markdown="">
- To validate the approach, we first did a screen for genes that regulate nonsense-mediated decay, or NMD.
- NMD is a well studied process that causes RNA decay when you have a premature stop codon upstream of an intron, as you can see in this schematic of the reporter we constructed and the qPCR measurement.
</aside>
            </section>
    



    
        <section >
            
            <h3>ReLiC identifies gene networks regulating NMD</h3>
<p><img src="./svg/nmd_lfc_fdr_scatter.svg" alt="" width="60%"></p>
<aside class="notes" data-markdown="">
<ul>
<li>When we performed ReLiC screens using the NMD reporter, we found that guides targeting known NMD factors such as UPF1, UPF2, and SMG1, readily increased the NMD reporter RNA levels.</li>
<li>We also found a large number of other hits involved in translation, and we did not find any hits that increased levels of the control reporter.</li>
</ul>
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/nmd_model_0.svg" alt=""></p>
<aside class="notes" data-markdown="">
- At first glance, finding translation factors is not surprising, because NMD is activated on a prematurely terminating ribosome.
- But, the translation factors that we found as hits, shown here in red, were all in the EIF2 initiation complex, as well as many amino acyl tRNA synthetases.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/nmd_model_1.svg" alt=""></p>
<aside class="notes" data-markdown="">
- This is consistent with old observations that EIF2 phosphorylation inhibits NMD.
- Many stresses such as amino acid starvation, ER stress, viral infection cause phosphorylation of EIF2 through well described kinases, which then leads to inhibition of translation initiation.
- But these stresses can also have pleiotropic effects on the cell, so we want to check if these hits really are driven by EIF2 phosphorylation.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/nmd_model_2.svg" alt=""></p>
<aside class="notes" data-markdown="">
- To do this, we used a small molecule called ISRIB, which inhibits EIF2 phosphorylation.
- Or we knocked down one of the kinases, GCN2, that senses uncharged tRNAs to phosphorylate EIF2.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/nmd_isrib_sggcn2.svg" alt=""></p>
<aside class="notes" data-markdown="">
<ul>
<li>These chemogenomic and modifier screens are really easy to do using ReLiC.</li>
<li>We take the Cas9-induced cells from the NMD screen, and treat them either with the small molecule ISRIB, or lentiviruses that deplete GCN2.</li>
<li>We can then look for fold changes of the NMD reporter due to the ISRIB or GCN2 perturbations.</li>
</ul>
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/nmd_isrib_sggcn2_results.svg" alt=""></p>
<aside class="notes" data-markdown="">
- We readily find that only the tRNA synthetase and ER stress-inducing hits are sensitive to ISRIB, and only the tRNA synthetase hits are sensitive to GCN2 depletion.
- This is consistent with the idea that these hits are driven by EIF2 phosphorylation, and shows the usefulness of ReLiC for modifier and chemogenomic screens.
- Note that these hits will be impossible to study using fluorescent reporters because knocking them down shuts down overall protein expression.
</aside>
            </section>
    



    
        <section >
            
            <h3>Can we identify regulators for normal mRNA metabolism using ReLiC?</h3>
<aside class="notes" data-markdown="">
- We then wanted to use ReLiC to identify regulators of a normal mRNA metabolic step.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="./svg/splicing_relic_design.svg" alt=""></p>
<aside class="notes" data-markdown="">
- As an example, we focused on splicing.
- Fluorescent reporters for splicing are quite challenging to build and validate, but it is really to do this with ReLiC.
- In fact, we can just take the RNA extracted from the control reporter from our NMD screen, and convert it to cDNA.
- We can then focus on either the normal mRNA species, or the intron retained or exon-skipped species, by simply changing where the forward PCR primer anneals during library construction.
- We can then look for guides that increase or decrease the levels of the intron retained or exon skipped species, relative to the normal mRNA species.
</aside>
            </section>
    



    
        <section >
            
            <h3>ReLiC screening for intron retention readily identifies splicing factors</h3>
<p><img src="./svg/gene_ntc_intron_lfc_fdr_scatter.svg" alt="" width="86%"></p>
<aside class="notes" data-markdown="">
- When we look at the hits that increase intron retention, we identify most of the known essential factors and complexes required for normal splicing, that are listed here.
</aside>
            </section>
    



    
        <section >
            
            <h3>Retained intron hits are distributed throughout the splicing cycle</h3>
<p><img src="./svg/splicing_cycle.svg" alt="" width="75%"></p>
<aside class="notes" data-markdown="">
- These hits are distributed throughout the entire splicing cycle, and are not enriched for any particular step.
</aside>
            </section>
    



    
        <section >
            
            <h3>ReLiC screen for exon skipping identifies factors involved in splice site selection</h3>
<p><img src="./svg/exon_skipping_scatter.png" alt="" width="86%">
<img src="./svg/exon_skipping_scatter.svg" alt="" width="86%"></p>
<aside class="notes" data-markdown="">
- By contrast, when we look at the hits that increase exon skipping, thy are centered on the SF3b and SF3a complexes that are known to be involved in splice site selection.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/hht_chx_ans_design.png" alt="">
<img src="svg/hht_chx_ans_design.svg" alt=""></p>
<aside class="notes" data-markdown="">
- I want to show one final example of a ReLiC screen that gave us an unexpected result.
- mRNA stability and translation are known to be coupled, but the factors that regulate this coupling are not well understood.
- So, we took a simple EYFP reporter, and performed a ReLiC chemogenomic screen to measure its RNA levels when we inhibited translation using three different drugs, harringtonine, anisomycin, and cycloheximide.
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/hht_chx_ans_results.png" alt="">
<img src="svg/hht_chx_ans_results.svg" alt=""></p>
<aside class="notes" data-markdown="">
<ul>
<li>The most unexpected hit came when we used harringtonine, which is an FDA-approved drug that is used for treating leukemia.</li>
<li>We found that the knocking out the protein GCN1L1, decreases RNA levels during harringtonine treatment but not during anisomycin or cycloheximide treatment. I wonâ€™t talk about these other hits here.</li>
</ul>
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/hht_model.png" alt="">
<img src="svg/hht_model.svg" alt=""></p>
<aside class="notes" data-markdown="">
<ul>
<li>GCN1L1, also just called GCN1, is a long snakelike protein that has been recently associated with colided ribosomes, and is thought to trigger mRNA decay.</li>
<li>As shown in the ribosome profiling data here, harringtonine itself arrests initiating ribosomes but does not affect elongating ribosomes. So, there should be no colliding ribosomes under harringtonine treatment.</li>
<li>So why does GCN1 show up as a hit here?</li>
</ul>
</aside>
            </section>
    



    
        <section >
            
            <p><img src="svg/hht_model_2.svg" alt=""></p>
<aside class="notes" data-markdown="">
<ul>
<li>Without showing some more follow up experiments, our current model is this:</li>
<li>We think that when harringtonine inhibits translation initiation on mRNAs with a single start codon with strong kozak, you do not have collisions.</li>
<li>But when you have two competing start codons with overlapping coding regions, then you get an elongating ribosome that initiated at an upstream start codon collide with a harringtonine-arrested initiating ribosome at a downstream start codon. We think this recruits GCN1, which then triggers mRNA decay.</li>
<li>We are currently testing this model using ribosome profiling and other approaches.</li>
</ul>

            </section>
    



    
        <section >
            
            <p><img src="svg/future_directions.svg" alt=""></p>
<aside class="notes" data-markdown="">
<ul>
<li>With that I will end my talk.</li>
<li>We are currently extending ReLiC to study RNA localization, perform deep mutational scanning of hits, as well as study viral and non-coding RNA biology.</li>
<li>Again, all this work was carried out by a talented grad student, Patrick Nugent, along with the help of a staff scientist Heungwon Park in the lab.</li>
<li>Thanks and I am happy to take any questions.</li>
</ul>
</aside>
            </section>
    


    </div>


  </div>

  <div class="line top"></div>
  <div class="line bottom"></div>
  <div class="line left"></div>
  <div class="line right"></div>

  <script src="libs/reveal.js/4.3.1/reveal.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/notes/notes.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/markdown/markdown.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/highlight/highlight.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/math/math.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/fullscreen/plugin.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/animate/plugin.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/animate/svg.min.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/anything/plugin.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/anything/Chart.min.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/anything/d3/d3.v3.min.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/anything/d3.patch.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/anything/d3/queue.v1.min.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/anything/d3/topojson.v1.min.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/anything/function-plot.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/customcontrols/plugin.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/embed-tweet/plugin.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/chart/chart.min.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/chart/plugin.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/verticator/verticator.js"></script>

<script src="libs/reveal.js/4.3.1/plugin/zoom/zoom.js"></script>
<script src="libs/reveal.js/4.3.1/plugin/search/search.js"></script>



<!--	<script src="libs/reveal.js/4.3.1/plugin/audio-slideshow/plugin.js"></script>  -->
<!--	<script src="libs/reveal.js/4.3.1/plugin/audio-slideshow/recorder.js"></script>-->
<!--	<script src="libs/reveal.js/4.3.1/plugin/audio-slideshow/RecordRTC.js"></script>-->

  

<script>
  const printPlugins = [
      RevealNotes,
      RevealHighlight,
      RevealMath.MathJax3,
      RevealAnimate,
      
			RevealEmbedTweet,
			RevealChart,
		];

		const plugins =  [...printPlugins,
		RevealZoom, 
		RevealSearch, 
				RevealMarkdown, 
				
				RevealFullscreen,
				RevealAnything,
				//RevealAudioSlideshow,
				//RevealAudioRecorder,
				
				// poll
				// question
				// seminar
				Verticator 
				 ]


		// Also available as an ES module, see:
		// https://revealjs.com/initialization/
		Reveal.initialize({
			controls: false,
      controlsTutorial: true,
      controlsLayout: 'bottom-right',
      controlsBackArrows: 'faded',
      progress: false,
      slideNumber: false,
      //#showSlideNumber "all" "print" "speaker"
      hash: true, //# hash: false,
      //# respondToHashChanges: true,
      //# history: false,
      keyboard: true,
      //#keyboardCondition: null,
      overview: true,
      center: true,
      touch: true,
      loop: false,
      rtl: false,
      //#navigationMode: 'default', linear grid
      shuffle: false,
      fragments: true,
      fragmentInURL: false,
      embedded: false,
      help: true,
      //#pause: true
      showNotes: false,
      autoPlayMedia: false, // TODO fix this to a nullable value
      //#preloadIframes: null. true false
      //#autoAnimate: true
      //#autoAnimateMatcher: null,
      //#autoAnimateEasing: 'ease',
      //autoAnimateDuration: 1.0,
      //#autoAnimateUnmatched: true
      //#autoAnimateStyles: []
      autoSlide: 0, // TODO fix this to a falseable value
      autoSlideStoppable: true,
      autoSlideMethod: '0',
      defaultTiming: 120,
      mouseWheel: false,
      //#previewLinks: false
      //#postMessage: true, // TODO : this can cause issues with the vscode api ???
      //#postMessageEvents: false,
      //#focusBodyOnPageVisibilityChange: true,
      transition: 'none',
      transitionSpeed: 'default',
      backgroundTransition: 'fade',
      //#pdfMaxPagesPerSlide: Number.POSITIVE_INFINITY,
      //#pdfSeparateFragments: true,
      //#pdfPageHeightOffset: -1,
      viewDistance: 3,
      //#mobileViewDistance: 2,
      display: 'block',
      //#hideInactiveCursor: true,
      //#hideCursorTime: 5000

      // Parallax Background
      parallaxBackgroundImage: '',
      parallaxBackgroundSize: '',
      parallaxBackgroundHorizontal: 0,
      parallaxBackgroundVertical: 0,

      //Presentation Size
      width: 960,
			height: 700,
			margin: 0.04,
      minScale: 0.2,
      maxScale: 2,
      disableLayout: false,

      audio: {
        prefix: 'audio/', // audio files are stored in the "audio" folder
        suffix: '.ogg', // audio files have the ".ogg" ending
        textToSpeechURL: null, // the URL to the text to speech converter
        defaultNotes: false, // use slide notes as default for the text to speech converter
        defaultText: false, // use slide text as default for the text to speech converter
        advance: 0, // advance to next slide after given time in milliseconds after audio has played, use negative value to not advance
        autoplay: false, // automatically start slideshow
        defaultDuration: 5, // default duration in seconds if no audio is available
        defaultAudios: true, // try to play audios with names such as audio/1.2.ogg
        playerOpacity: 0.05, // opacity value of audio player if unfocused
        playerStyle: 'position: fixed; bottom: 4px; left: 25%; width: 50%; height:75px; z-index: 33;', // style used for container of audio controls
        startAtFragment: false, // when moving to a slide, start at the current fragment or at the start of the slide
      },
      
			customcontrols: {
					controls: [
      						{
						  id: 'toggle-overview',
						  title: 'Toggle overview (O)',
						  icon: '<i class="fa fa-th"></i>',
						  action: 'Reveal.toggleOverview();'
						}
						
				]
			},
			chart: {
					defaults: { 
						color: 'lightgray', // color of labels
						scale: { 
							beginAtZero: true, 
							ticks: { stepSize: 1 },
							grid: { color: "lightgray" } , // color of grid lines
						},
					},
					line: { borderColor: [ "rgba(20,220,220,.8)" , "rgba(220,120,120,.8)", "rgba(20,120,220,.8)" ], "borderDash": [ [5,10], [0,0] ] }, 
					bar: { backgroundColor: [ "rgba(20,220,220,.8)" , "rgba(220,120,120,.8)", "rgba(20,120,220,.8)" ]}, 
					pie: { backgroundColor: [ ["rgba(0,0,0,.8)" , "rgba(220,20,20,.8)", "rgba(20,220,20,.8)", "rgba(220,220,20,.8)", "rgba(20,20,220,.8)"] ]},
					radar: { borderColor: [ "rgba(20,220,220,.8)" , "rgba(220,120,120,.8)", "rgba(20,120,220,.8)" ]}, 
			},
			math: {
				mathjax: 'https://cdn.jsdelivr.net/gh/mathjax/mathjax@2.7.8/MathJax.js',
				config: 'TeX-AMS_HTML-full',
				// pass other options into `MathJax.Hub.Config()`
				TeX: { Macros: { RR: "{\\bf R}" } }
				},
				anything: [ 
				{
		className: "plot",
		defaults: {width:500, height: 500, grid:true},
		initialize: (function(container, options){ options.target = "#"+container.id; functionPlot(options) })
	 },
	 {
		className: "chart",  
		initialize: (function(container, options){ container.chart = new Chart(container.getContext("2d"), options);  })
	 },
	 {
		className: "anything",
		initialize: (function(container, options){ if (options && options.initialize) { options.initialize(container)} })
	 },
					],
			// Learn about plugins: https://revealjs.com/plugins/
			plugins: (window.location.search.match(/print-pdf/gi) ? printPlugins : plugins ) 
		});
			


	    // Change chalkboard theme : 
		function changeTheme(input) {
			var config = {};
			config.theme = input.value;
			Reveal.getPlugin("RevealChalkboard").configure(config);
			input.blur();
		}

		// // Handle the message inside the webview
        // window.addEventListener('message', event => {

        //     const message = event.data; // The JSON data our extension sent

        //     switch (message.command) {
        //         case 'refactor':
        //             Reveal.toggleHelp();
        //     }
        // });

		if (window.location.search.match(/print-pdf-now/gi)) {
      		setTimeout(() => {
				window.print();
			  }, 2500);
			
    }
</script>

</body>

</html>